tests/llmcompressor/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -456,7 +455,7 @@ def test_sparse_24_compressor_is_lossless(model_stub, recipe, sparse_format, tmp
     splits = {"calibration": "train[:10%]"}
     empty_model = AutoModelForCausalLM.from_pretrained(model_stub, torch_dtype="auto")
 
-    oneshot(
+    model = oneshot(
```

Comment:
so `oneshot` both modifies the model passed in and returns it here? That might confuse users

Reply:
model meaning PretrainedModel? What do you mean model here?

Reply:
the main flow i've seen is user creates model
```python
model = AutoModelForCausalLM....
```
and passes that into oneshot
```python
oneshot(model, ds, recipe)
```
and then access the compressed model which has been modified in place
```
model.save_pretrained....
```
rather than doing `model = oneshot(....)`. I didn't realize `oneshot` also returned the model. I think this makes more sense, I just didn't realize this worked.

Reply:
i'll mark this as resolved, i didn't realize we had `def oneshot(**kwargs) -> PreTrainedModel:`